pom.xml

```diff
@@ -81,7 +81,7 @@
     <dependency>
       <groupId>org.apache.maven.shared</groupId>
       <artifactId>maven-filtering</artifactId>
-      <version>3.2.0</version>
+      <version>3.3.2</version>
```

Comment:
I think we had to downgrade this dependency multiple times; there was a problem only occurring the other day when re-integrating (e.g. in demo-projects) build.

Reply:
Ok I will try it. Otherwise revert it.

Reply:
Seems to work:
https://jenkins.ivyteam.io/job/demo-projects/job/master/